### ISSUE TITLE
Package Structure is weird

### ISSUE THREAD

**Issue by Sollace** [BlazeLoader/BlazeLoader#28]
I've started looking at the code again and one thing I notice that I'm not really sure is intentional or not.

A lot of the packages got com.blazeloader.api.api.<whatever>.java

There are two .api's in the path which just seems... odd. I'm going to leave it like that for now and do my pull request with what I've already done.

**Comment by warriordog** [comment]
Yeah I've thought about fixing that, but i couldn't decide what to do.  I will probably switch the first .api to .bl, but probably not until another major update.

**Comment by Sollace** [comment]
Actually I just came back to add another weirdness. It also goes .api.client.api.*

I suggest just changing it to 1 .api

.api.client.api should just be .client.api or .api.client.

I could have a go at reorganizing it on my repo and you can have a look to see how it turns out.

**Comment by warriordog** [comment]
You can if you'd like, otherwise I can refactor it a bit when I am home.  I wrote it like that when I was going through a super-packaging phase :)

**Comment by Sollace** [comment]
Meh, I'll see. Rather leave it for now since I have a huge pull request to submit.

**Comment by warriordog** [comment]
@Sollace 
This was fixed in your recent commit, right?

**Comment by Sollace** [comment]
@warriordog Yes, in the moving-all-the-things PR. I don't know how to close this issue though.

**Comment by warriordog** [comment]
I can close it when I am on PC.
